roles/yoda_portal/tasks/main.yml

```diff
@@ -1,22 +1,42 @@
 ---
 # copyright Utrecht University
 
+- name: Ensure yodadeployment user exists
+  user:
+    name: "{{ yoda_deployment_user }}"
+
+
+- name: Ensure yodadeployment has access to /var/www/yoda
+  file:
+    path: /var/www/yoda
+    state: directory
+    owner: "{{ yoda_deployment_user }}"
+    group: "{{ yoda_deployment_user }}"
+    mode: 0711
+
+
 - name: Checkout Yoda portal
+  become_user: "{{ yoda_deployment_user }}"
+  become: yes
   git:
     repo: "https://github.com/UtrechtUniversity/yoda-portal.git"
     dest: "/var/www/yoda"
-    version: "{{ yoda_portal_version }}"
+    version: "pyportal-ansible"
```

Comment:
Should probably be {{ yoda_portal_version }} ?

Reply:
Yes, updated (reverted) it to {{ yoda_portal_version }}